### PR TITLE
Update Spanish translation for 1.21.4

### PR DESCRIPTION
**Pull request by xoascf**

### CODE REVIEW

File: resources/es/cemu.po [hunk 1]
```diff
@@ -634,15 +924,15 @@ msgstr ""
 
 #: GeneralSettings2.cpp
 msgid "Discord Presence"
-msgstr "Rich Presence de Discord"
+msgstr "Presencia enriquecida de Discord"
```
Comment (Fs00) [inline resources/es/cemu.po:927]: It seems like that the official name of this feature is "Rich Presence" in Spanish too, see https://discord.com/rich-presence. Therefore I'm not sure whether its name should be translated

File: resources/es/cemu.po [hunk 2]
```diff
@@ -2341,7 +2680,7 @@ msgstr ""
 
 #: TitleManager.cpp
 msgid "Irrevocable delete the save entry "
-msgstr "Elimina irrevocablemente la entrada de partida guardada"
+msgstr "Elimina irrevocablemente la entrada de partida guardada "
```
Comment (Fs00) [inline resources/es/cemu.po:2683]: The trailing whitespace is a typo of the original string, you can remove it if you want

File: resources/es/cemu.po [hunk 3]
```diff
@@ -2151,19 +2490,19 @@ msgstr "&Ir al volcado"
...
 
 #: SaveImportWindow.cpp
 msgid "Select a zipped save file"
-msgstr "Seleccione un archivo de guardado comprimido"
+msgstr "Seleccione un archivo de partida guardada comprimida"
 
 #: SaveImportWindow.cpp
 msgid "Import save entry {}"
-msgstr "Importar entrada de la partida guardada {}"
+msgstr "Entrada de partida guardada para importar {}"
```
Comment (Fs00) [inline resources/es/cemu.po:2505]: Could you tell me where this string is used in the application and why did you change it like that? Because I translated it like you did before, but honestly I'm not sure what is being put in place of `{}`.

File: resources/es/cemu.po [hunk 4]
```diff
@@ -194,6 +310,180 @@ msgstr "Reiniciar"
...
+msgid "Generated checksum of {} game files"
+msgstr "Suma de comprobación de {} archivos de juego generada"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}"
+msgstr "Leyendo imagen de juego: {}"
+
+#: ChecksumTool.cpp
+msgid "Reading game image: {}/{}kb"
+msgstr "Leyendo imagen de juego: {} de {} KiB"
```
Comment (Fs00) [inline resources/es/cemu.po:477]: Are you sure that it's "KiB" and not "KB"? Because they are actually different units.

**Review by Fs00 — COMMENTED** [review]
Looks good, just a few minor things that probably should be addressed (and a question for you🙂).

File: resources/es/cemu.po (hunk 1 above)
Comment (xoascf) [reply resources/es/cemu.po:927]: Yes, I hope Discord will correct and update that term.

File: resources/es/cemu.po (hunk 3 above)
Comment (xoascf) [reply resources/es/cemu.po:2505]: When importing a file to compare the checksum

File: resources/es/cemu.po (hunk 4 above)
Comment (xoascf) [reply resources/es/cemu.po:477]: Sure, try to dump a game into the WUD format, and then get the file size (KiB is the correct term, MS Windows incorrectly uses "KB" when referring to that size, macOS or any "Linux-based operating system" has already fixed that issue).

File: resources/es/cemu.po (hunk 2 above)
Comment (xoascf) [reply resources/es/cemu.po:2683]: Sure? well, I left it like that because they haven't corrected it yet, and I thought it was for some reason...

File: resources/es/cemu.po (hunk 3 above)
Comment (Fs00) [reply resources/es/cemu.po:2505]: Ok, thanks!

File: resources/es/cemu.po (hunk 4 above)
Comment (Fs00) [reply resources/es/cemu.po:477]: Ok! I don't have any WUD images at hand, so that's why I was asking

File: resources/es/cemu.po (hunk 2 above)
Comment (Fs00) [reply resources/es/cemu.po]: Yep, that's the tooltip for the "Delete" button of saves

File: resources/es/cemu.po (hunk 1 above)
Comment (Fs00) [reply resources/es/cemu.po:927]: My idea is that we should stick with officially translated names as much as we can. Obviously, if you deem the official name incorrect and your translation is more commonly used, I have no problems merging it.

**Comment by Fs00** [comment]
Thanks!
